src/SurveyService.Mappers/Db/Interfaces/IDbQuestionMapper.cs

```diff
@@ -0,0 +1,10 @@
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Mappers.Db.Interfaces;
+
+public interface IDbQuestionMapper
+{
+  DbQuestion Map(CreateQuestionRequest dbQuestionRequest, Guid? groupId);
```

Comment:
@vorobevamv 
I thought that CreateQuestionRequest doesn't include GroupId because, if we create just one question without a group we can't pass GroupId, so we set it to null in Mapper (we just can't pass GroupId). If we create questions while creating group we can't know GroupId as it doesn't exist yet.
Did I manage to clarify the situation?

Reply:
Lev is right. GroupId in CreateQuestionRequest can be null, if it is a single question